supervisor/api/__init__.py

```diff
@@ -287,6 +287,7 @@ def _register_supervisor(self) -> None:
                 web.get(
                     "/supervisor/available_updates", api_supervisor.available_updates
                 ),
+                web.post("/supervisor/refresh_updates", api_supervisor.reload),
```

Comment:
Change path since it refreshes all updates.
```suggestion
                web.post("/refresh_updates", api_supervisor.reload),
```